Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -91,15 +125,36 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
                     Parse();
                 }
             }
+            else if (WizardEventArgs.EventType == EventType.LeavingForNextPage)
+            {
+                if (AddAPIModelWizard.DeltaModelsList != null && AddAPIModelWizard.DeltaModelsList.Count > 0)
```

Comment:
better trigger flag on Wizard level called "CompareWasDone" which will set to true in case user clicked on "Compare" button and use it where ever needed to know if compare done or not